src/orchard/misc.clj

```diff
@@ -3,8 +3,19 @@
    [clojure.java.io :as io]
    [clojure.string :as str]))
 
+(def os-name
```

Comment:
I think we can put the os-* in the new namespace as well. We'll just keep the existing functions here for compatibility and deprecate them.